selfdrive/car/honda/carstate.py

```diff
@@ -207,6 +180,13 @@ def __init__(self, CP):
     self.gearbox_msg = "GEARBOX"
     if CP.carFingerprint == CAR.ACCORD and CP.transmissionType == TransmissionType.cvt:
       self.gearbox_msg = "GEARBOX_15T"
+    # message containing CAR_GAS, if any
+    self.gas_pedal_msg = "GAS_PEDAL_2"
+    if self.CP.carFingerprint in (CAR.CRV, CAR.CRV_EU, CAR.ODYSSEY, CAR.ACURA_RDX, CAR.RIDGELINE, CAR.PILOT, CAR.PILOT_2019, CAR.ODYSSEY_CHN):
+      # crv et al don't include cruise control
+      self.gas_pedal_msg = None
```

Comment:
note: I added CAR.PILOT to this case

The original code was adding `("CAR_GAS", "GAS_PEDAL_2", 0)` to signals, but then had a comment on the `GAS_PEDAL_2` check that the message wasn't available on some Pilots.  If that did happen, it would mean CarState.gas would always be 0, which sounds problematic.  So it seems better to put PILOT entirely in the "GAS_PEDAL not available" group, so that `gas` will be derived from `PEDAL_GAS`.

Reply:
now irrelevant since all models are using PEDAL_GAS